### PR TITLE
Add Ubuntu 16.10 platforms and test fixtures

### PR DESCRIPTION
**Pull request by ScottGarman**
Added Ubuntu 16.10 platforms for i386 and x86_64. The test fixtures were created by running rake generate:fixtures.

### CODE REVIEW

File: test/fixtures/generated/default/ubuntu1610-32l [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      platform: ubuntu-16.10-i386
+      template: ubuntu-1610-i386
+      roles:
+      - agent
+      - classifier
+  CONFIG:
+    nfs_server: none
+    consoleport: 443
+    pooling_api: http://vmpooler.delivery.puppetlabs.net/
+expected_exception:
```
Comment (kurtwall) [inline test/fixtures/generated/default/ubuntu1610-32l]: Is this boilerplate or is an entry for 32-bit systems necessary? I didn't realize we were still testing 32-bit *NIX platforms.

Comment (ScottGarman) [inline test/fixtures/generated/default/ubuntu1610-32l]: To my understanding, we've dropped support for 32-bit puppet servers, but where the puppet-agent is concerned, we are still supporting and testing 32-bit platforms.

Comment (McdonaldSeanp) [inline test/fixtures/generated/default/ubuntu1610-32l]: +1, that is also my undestanding @ScottGarman

**Comment by McdonaldSeanp** [comment]
👍 LGTM

**Comment by geoffnichols** [comment]
I think this looks good, but pinging @nwolfe for review.

**Comment by johnduarte** [comment]
I realize this is after the fact, but Ubuntu 16.10 was classified as a Tier 3 platform and is not expected to receive coverage in CI. As such, this addition not necessary.

Ref: https://tickets.puppetlabs.com/browse/QA-2734

**Pull request merged**